Scripts/script-SetSeverityByScore.yml

```diff
@@ -47,6 +47,7 @@ tags:
 - url
 - ip
 - hash
+comment: "-"
```

Comment:
@asafshen

Reply:
@liorkol I would like to get your help on that..
(feeling that writing 'Set incident severity by existing indicators' is not enough)